utils/cpuload/scheddebug/scheddebug.go

```diff
@@ -142,9 +178,52 @@ func (self *SchedReader) allowErrorLogging() bool {
 	return false
 }
 
+func GetSelfCgroup() (string, error) {
```

Comment:
We call it just once and cache it here. Wanted to keep cAdvisor dependencies on utils at a minimum.